scripts/migration.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+# This script manages the migration of your content from version 1.0 to 1.1
+#
+# To use this script, you need to:
+# 1. Have run make deploy to create a new IA resource group
+# 2. Copy over the inf_output.json file from the output folder of the 1.0 deployment
+#    to the scripts folder of the 1.1 deployment and rename if to in_output_old.json
+# 3. run make run-migration to perfomr the migration
+#
+# The script will:
+# 1. Read required values from in_output.json
+# 2. Read required values from in_output_old.json
+# 3. Read required secrets from azure keyvault
+# 4. Migrate Cosmos DB tags from the old tags container and database to the
+
+
+from pyfiglet import Figlet
+import json
+import os
+from azure.cosmos import CosmosClient, PartitionKey, exceptions
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.secrets import SecretClient
+
+f = Figlet()
+print(f.renderText('Migration from 1.0 to 1.1'))
+print("Current Working Directory:", os.getcwd())
+# Read required secrets from azure keyvault
+credential = DefaultAzureCredential()
+
+
+# *************************************************************************
+# Read required NEW values from in_output.json & new key vault
```

Comment:
We don't technically have old and new databases anymore. we are just merging the documents from the tags container into the status container. Can we update the logic to reflect this?